src/pages/contract/_components/ReadContractCodeRow.tsx

```diff
@@ -0,0 +1,172 @@
+import GradientCardContainer from "@components/commons/GradientCardContainer";
+import { Dispatch, SetStateAction, useEffect, useRef, useState } from "react";
+import { FiCopy, FiShare2, FiMaximize, FiMinimize } from "react-icons/fi";
+import { sleep } from "shared/sleep";
+import { CodeBlock } from "react-code-blocks";
+import clsx from "clsx";
+import Tooltip from "@components/commons/Tooltip";
+
+export default function ReadContractCodeRow({
```

Comment:
i think `..Row` is too tightly coupled with the representation of UI
```suggestion
export default function ReadContractCodeFile({
```

Reply:
renamed to ReadContractCodeFile and renamed WriteContractRow to WriteContractInputItem